src/script/conversation/ConversationRepository.js

```diff
@@ -2083,15 +2122,11 @@ z.conversation.ConversationRepository = class ConversationRepository {
    * @returns {Promise} Resolves after sending the location
    */
   sendLocation(conversationEntity, longitude, latitude, name, zoom) {
-    const genericMessage = new z.proto.GenericMessage(z.util.createRandomUuid());
-    const protoLocation = new z.proto.Location(
-      longitude,
-      latitude,
-      name,
-      zoom,
-      this.expectReadReceipt(conversationEntity)
-    );
-    genericMessage.set(z.cryptography.GENERIC_MESSAGE_TYPE.LOCATION, protoLocation);
+    const protoLocation = new Location(longitude, latitude, name, zoom, this.expectReadReceipt(conversationEntity));
```

Comment:
I believe the `Location` constructor should also take an object as a single parameter, isn't it?

Reply:
Yes of course, good find!